extensions/git/src/repository.ts

```diff
@@ -727,6 +727,12 @@ export class Repository implements Disposable {
 		this.updateCommitTemplate();
 	}
 
+	updateInputBox() {
+		const headless: Boolean = this.headLabel === '';
+		// '{0}' will be replaced by the corresponding key-command later in the process, which is why it needs to stay.
+		this._sourceControl.inputBox.placeholder = localize(headless ? 'commitMessage' : 'commitMessageWithHeadLabel', "Message ({0} to commit{1})", "{0}", headless ? 'this.headLabel' : ' to ' + this.headLabel);
```

Comment:
As @burknator [mentions](https://github.com/microsoft/vscode/issues/73644#issuecomment-500998333): this _breaks localization of the placeholder_.